recipes/skaware/s6.inc

```diff
@@ -0,0 +1,37 @@
+DESCRIPTION = "s6 is a small suite of programs for UNIX, designed to allow process supervision (a.k.a service supervision), in the line of daemontools and runit, as well as various operations on processes and daemons."
+LICENSE = "ISC"
+
+SRC_URI = "http://skarnet.org/software/${PN}/${PN}-${PV}.tar.gz"
+
+inherit c make
+
+DEPENDS += "libskarnet libexecline"
+
+do_configure() {
```

Comment:
Maybe, but could we postpone that to a future action so we can get this merged for 6.0?